src/lib/popper.mli

```diff
@@ -309,6 +327,33 @@ module Sample : sig
     val lower : string t
   end
 
+  module Bytes : sig
+    (** [of_length n] is a sample that always produces [bytes] of fixed size
+        [n]. *)
+    val of_length : int -> bytes t
+
+    (** [range min max] is a sample that always produces a [bytes] of size [n],
+        where [min <= n < max]. *)
+    val range : int -> int -> bytes t
+
+    (** [alpha_numeric] is a sample that produces [bytes] containing
+        alhpa-numeric characthers. *)
+    val alpha_numeric : bytes t
+
+    (** [numeric] is a sample that produces [bytes] containing numbers *)
+    val numeric : bytes t
+
+    (** [alpha] a sample that produces [bytes] containing characters a-z, both
+        upper and lower case. *)
+    val alpha : bytes t
+
+    (** [upper] is sample that always produces upper-case characters. *)
+    val upper : bytes t
+
+    (** [upper] is sample that always produces lower-case characters. *)
```

Comment:
```suggestion
    (** [lower] is sample that always produces lower-case characters. *)
```

Reply:
@bikallem do you think it makes sense to expose all these combinators for `bytes`? When would you use `bytes` rather than `string`?

Reply:
Good question. From my experience, usually when consuming networking APIs,  `bytes` is used frequently to expose input/output to the APIs rather than `string`. In those scenarios it is quite convenient to work straight in `bytes` rather than having to convert to string and then use the String module functions. 

Additionally, OCaml stdlib String and Bytes seem to mirror each other's API; so it is perhaps "good" to maintain the OCaml convention.

Reply:
Ok, makes sense. Happy to keep.